browser/src/control/jsdialog/Widget.ColorPicker.ts

```diff
@@ -21,6 +21,14 @@ interface ColorPaletteWidgetData {
 	command: string;
 }
 
+interface ThemeColor {
+	Value: string;
+	Name: string;
```

Comment:
If theme colors already have name in own entry, cant we have similar for non-theme colors? in other palettes?